tools/inference_engine_tests/test_cases/test_model.cpp

```diff
@@ -151,7 +151,7 @@ TEST(ModelTest, MatMul_fused_activation)
     // ask model for output size (we know that there has to be 1 output in this test case)
```

Comment:
Please add new test case to this PR which tests multiple layers which can be fused. For example:
Port -> Activation -> Activation -> Activation -> Activation